source/_lovelace/glance.markdown

```diff
@@ -24,22 +24,12 @@ type:
   type: string
 entities:
   required: true
-  description: "A list of entity IDs or an `entity` object."
+  description: "A list of entity IDs or a `entity` objects."
   type: list
-  keys:
-    entity:
-      required: true
-      description: "The `entity_id` to show."
-      type: string
-    name:
-      required: true
-      description: "A name for `the entity_id`."
-      type: string
```

Comment:
According to sample below those are valid keys. From my point of view those needs to be documented.

Reply:
The keys there are optional. Yep, I will add a new setion for this